src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.ts

```diff
@@ -219,6 +219,7 @@ export class OsdListComponent implements OnInit {
           }
         }
       },
+      { prop: 'tree.device_class', name: this.i18n('Class') },
```

Comment:
```suggestion
      { 
        prop: 'tree.device_class',
        name: this.i18n('Class'), 
        flexGrow: 1,
        cellTransformation: CellTemplate.badge,
        customTemplateConfig: {
          map: {
            hdd: { value: 'HDD', class: 'badge-hdd' },
            ssd: { value: 'SSD', class: 'badge-ssd' }
          }
        }
      },
```

Reply:
Maybe we should label the column `Type` like it is done in the Inventory page. @bk201

Reply:
Yes, but maybe just for a better look.

Initially I was wondering if we should call the column in Inventory (`type`) and this one (`class`) to the same name. But turn out It's not right because CRUSH class can be defined by users. Types in inventory are likely fixed and hard-coded from orchestrator layer.

Reply:
@votdev @bk201 I've now renamed the column to "**Device class**" as it seems to be the most appropriate name based on [this article](https://ceph.io/community/new-luminous-crush-device-classes), and based on the commands that are used to manage it:

```
$ ceph osd crush get-device-class osd.2
$ ceph osd crush rm-device-class osd.2 
$ ceph osd crush set-device-class ssd osd.2
```

I've also applied the suggested `badges` style, except the uppercase transformation because this value can be edited by the user, and it's case sensitive:

![Screenshot from 2019-12-10 10-24-49](https://user-images.githubusercontent.com/14297426/70522406-f7ef6000-1b38-11ea-8ce9-17e1b817c8ee.png)